tests/unit/test_lsr_role2collection.py

```diff
@@ -0,0 +1,413 @@
+        shutil.rmtree(test_role_path)
+        # case 3. tests/roles/linux-system-roles.systemrole -> roles/systemrole
+        #         tests/roles/some_symlinks
+        self.create_test_link(test_role_path, link, role_path, True)
+        extralink = "extralink"
+        self.create_test_link(test_role_path, extralink, role_path, True)
+        cleanup_symlinks(test_path, rolename)
+        self.check_test_link(test_role_path, True)
+        self.check_test_link(test_role_path / link, False)
+        self.check_test_link(test_role_path / extralink, True)
+        shutil.rmtree(test_role_path)
+
+    def test_import_replace(self):
+        module_name = "util0"
+        src_module_dir = Path(src_path) / rolename / "module_utils" / module_name
+        src_module_dir.mkdir(parents=True, exist_ok=True)
+        dest_module_dir_core = (
+            Path(dest_path)
+            / "ansible_collections"
+            / namespace
+            / collection_name
+            / "plugins"
+            / "module_utils"
+        )
+        dest_module_dir = dest_module_dir_core / module_name
+        dest_module_dir.mkdir(parents=True, exist_ok=True)
+        input = bytes(
+            "import os\nimport ansible.module_utils.{0}\nimport re\n".format(
+                module_name
+            ),
+            "utf-8",
+        )
+        expected = bytes(
+            "import os\nimport ansible_collections.{0}.{1}.plugins.module_utils.{2}\nimport re\n".format(
+                namespace, collection_name, module_name
+            ),
+            "utf-8",
+        )
+        IMPORT_RE = re.compile(
+            br"(\bimport) (ansible\.module_utils\.)(\S+)(.*)$", flags=re.M
+        )
+        output = IMPORT_RE.sub(import_replace, input)
+        self.assertEqual(output, expected)
+        shutil.rmtree(src_module_dir)
+        shutil.rmtree(dest_module_dir)
+
+    def test_from_replace(self):
+        module_name = "util0"
+        src_module_dir = Path(src_path) / rolename / "module_utils" / module_name
+        src_module_dir.mkdir(parents=True, exist_ok=True)
+        dest_module_dir_core = (
+            Path(dest_path)
+            / "ansible_collections"
+            / namespace
+            / collection_name
+            / "plugins"
+            / "module_utils"
+        )
+        dest_module_dir = dest_module_dir_core / module_name
+        dest_module_dir.mkdir(parents=True, exist_ok=True)
+
+        test_files = ["test0", "test1", "test2", "test3", "__init__"]
+        for f in test_files:
+            fpath = dest_module_dir / (f + ".py")
+            fpath.touch(exist_ok=True)
+        gather_module_utils_parts(dest_module_dir_core)
+
+        input = bytes(
+            "# pylint: disable=import-error, no-name-in-module\n{0}.basic import AnsibleModule\n{0}.{1} import {2}\n{0}.{1} import MyError\n{0}.{1}.{3} import (\n    ArgUtil,\n    ArgValidator_ListConnections,    ValidationError,\n)\n{0}.{1}.{4} import Util\n{0}.{1} import {5}\n".format(
```

Comment:
You should use a multiline string with `textwrap.dedent` as in https://github.com/linux-system-roles/auto-maintenance/pull/13/files#diff-30cbdea1a4bfc877c6034e8e28d83700R478

Reply:
Thanks! Done.